src/common/components/Navbar.vue

```diff
@@ -319,6 +313,7 @@ export default {
           this.wallet.address
         )
         this.setWalletBalance(balanceNummber)
+        this.polkadotBalance = Number(this.walletBalance).toFixed(3)
```

Comment:
Instead of using `toFixed` everytime you call `walletBalance` maybe just keep a 3 fixed decimal in the `walletBalance` in the first place. 

So, essentially, edit in the Vue store instead.

Reply:
good idea, ok sure I will try to update the code 👍🏼